tests/by-util/test_ls.rs

```diff
@@ -2021,3 +2021,38 @@ fn test_ls_path() {
         .run()
         .stdout_is(expected_stdout);
 }
+
+#[test]
+fn test_ls_size() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+
+    at.touch("test-1");
+    at.append(
+        "test-1",
+        &std::iter::repeat("a\n").take(123456).collect::<String>(),
```

Comment:
clippy isn't happy here :)

Reply:
I don't see it?

```
hbina@DESKTOP-2D0I5MJ:~/git/coreutils$ cargo clippy --all-targets -- --cfg true
   Compiling coreutils v0.0.6 (/home/hbina/git/coreutils)
   Compiling uu_factor v0.0.6 (/home/hbina/git/coreutils/src/uu/factor)
   Compiling uucore_procs v0.0.5 (/home/hbina/git/coreutils/src/uucore_procs)
    Checking uucore v0.0.8 (/home/hbina/git/coreutils/src/uucore)
    Checking uu_base32 v0.0.6 (/home/hbina/git/coreutils/src/uu/base32)
    Checking uu_printf v0.0.6 (/home/hbina/git/coreutils/src/uu/printf)
    Checking uu_mktemp v0.0.6 (/home/hbina/git/coreutils/src/uu/mktemp)
    Checking uu_shuf v0.0.6 (/home/hbina/git/coreutils/src/uu/shuf)
    Checking uu_env v0.0.6 (/home/hbina/git/coreutils/src/uu/env)
    Checking uu_printenv v0.0.6 (/home/hbina/git/coreutils/src/uu/printenv)
    Checking uu_csplit v0.0.6 (/home/hbina/git/coreutils/src/uu/csplit)
    Checking uu_cp v0.0.6 (/home/hbina/git/coreutils/src/uu/cp)
    Checking uu_head v0.0.6 (/home/hbina/git/coreutils/src/uu/head)
    Checking uu_tail v0.0.6 (/home/hbina/git/coreutils/src/uu/tail)
    Checking uu_join v0.0.6 (/home/hbina/git/coreutils/src/uu/join)
    Checking uu_df v0.0.6 (/home/hbina/git/coreutils/src/uu/df)
    Checking uu_cut v0.0.6 (/home/hbina/git/coreutils/src/uu/cut)
    Checking uu_mkdir v0.0.6 (/home/hbina/git/coreutils/src/uu/mkdir)
    Checking uu_dirname v0.0.6 (/home/hbina/git/coreutils/src/uu/dirname)
    Checking uu_du v0.0.6 (/home/hbina/git/coreutils/src/uu/du)
    Checking uu_echo v0.0.6 (/home/hbina/git/coreutils/src/uu/echo)
    Checking uu_split v0.0.6 (/home/hbina/git/coreutils/src/uu/split)
    Checking uu_tsort v0.0.6 (/home/hbina/git/coreutils/src/uu/tsort)
    Checking uu_paste v0.0.6 (/home/hbina/git/coreutils/src/uu/paste)
    Checking uu_sum v0.0.6 (/home/hbina/git/coreutils/src/uu/sum)
    Checking uu_relpath v0.0.6 (/home/hbina/git/coreutils/src/uu/relpath)
    Checking uu_ptx v0.0.6 (/home/hbina/git/coreutils/src/uu/ptx)
    Checking uu_truncate v0.0.6 (/home/hbina/git/coreutils/src/uu/truncate)
    Checking uu_pwd v0.0.6 (/home/hbina/git/coreutils/src/uu/pwd)
    Checking uu_mv v0.0.6 (/home/hbina/git/coreutils/src/uu/mv)
    Checking uu_fmt v0.0.6 (/home/hbina/git/coreutils/src/uu/fmt)
    Checking uu_dircolors v0.0.6 (/home/hbina/git/coreutils/src/uu/dircolors)
    Checking uu_uniq v0.0.6 (/home/hbina/git/coreutils/src/uu/uniq)
    Checking uu_ln v0.0.6 (/home/hbina/git/coreutils/src/uu/ln)
    Checking uu_more v0.0.6 (/home/hbina/git/coreutils/src/uu/more)
    Checking uu_test v0.0.6 (/home/hbina/git/coreutils/src/uu/test)
    Checking uu_ls v0.0.6 (/home/hbina/git/coreutils/src/uu/ls)
    Checking uu_cat v0.0.6 (/home/hbina/git/coreutils/src/uu/cat)
    Checking uu_cksum v0.0.6 (/home/hbina/git/coreutils/src/uu/cksum)
    Checking uu_seq v0.0.6 (/home/hbina/git/coreutils/src/uu/seq)
    Checking uu_false v0.0.6 (/home/hbina/git/coreutils/src/uu/false)
    Checking uu_expand v0.0.6 (/home/hbina/git/coreutils/src/uu/expand)
    Checking uu_rmdir v0.0.6 (/home/hbina/git/coreutils/src/uu/rmdir)
    Checking uu_comm v0.0.6 (/home/hbina/git/coreutils/src/uu/comm)
    Checking uu_date v0.0.6 (/home/hbina/git/coreutils/src/uu/date)
    Checking uu_realpath v0.0.6 (/home/hbina/git/coreutils/src/uu/realpath)
    Checking uu_shred v0.0.6 (/home/hbina/git/coreutils/src/uu/shred)
    Checking uu_sleep v0.0.6 (/home/hbina/git/coreutils/src/uu/sleep)
    Checking uu_wc v0.0.6 (/home/hbina/git/coreutils/src/uu/wc)
    Checking uu_true v0.0.6 (/home/hbina/git/coreutils/src/uu/true)
    Checking uu_basename v0.0.6 (/home/hbina/git/coreutils/src/uu/basename)
    Checking uu_fold v0.0.6 (/home/hbina/git/coreutils/src/uu/fold)
    Checking uu_link v0.0.6 (/home/hbina/git/coreutils/src/uu/link)
    Checking uu_rm v0.0.6 (/home/hbina/git/coreutils/src/uu/rm)
    Checking uu_expr v0.0.6 (/home/hbina/git/coreutils/src/uu/expr)
    Checking uu_yes v0.0.6 (/home/hbina/git/coreutils/src/uu/yes)
    Checking uu_tac v0.0.6 (/home/hbina/git/coreutils/src/uu/tac)
    Checking uu_readlink v0.0.6 (/home/hbina/git/coreutils/src/uu/readlink)
    Checking uu_nl v0.0.6 (/home/hbina/git/coreutils/src/uu/nl)
    Checking uu_tee v0.0.6 (/home/hbina/git/coreutils/src/uu/tee)
    Checking uu_hashsum v0.0.6 (/home/hbina/git/coreutils/src/uu/hashsum)
    Checking uu_unexpand v0.0.6 (/home/hbina/git/coreutils/src/uu/unexpand)
    Checking uu_pr v0.0.6 (/home/hbina/git/coreutils/src/uu/pr)
    Checking uu_sort v0.0.6 (/home/hbina/git/coreutils/src/uu/sort)
    Checking uu_od v0.0.6 (/home/hbina/git/coreutils/src/uu/od)
    Checking uu_tr v0.0.6 (/home/hbina/git/coreutils/src/uu/tr)
    Checking uu_base64 v0.0.6 (/home/hbina/git/coreutils/src/uu/base64)
    Finished dev [unoptimized + debuginfo] target(s) in 7.23s
```

Reply:
Running `cargo clippy --all-targets` should do it.

Reply:
Nothing

```
coreutils on  hbina-ls-implement-size-display [?] is 📦 v0.0.6 via 🦀 v1.53.0 took 2s 
❯ cargo clippy --all-targets
   Compiling coreutils v0.0.6 (/home/hbina/git/coreutils)
    Checking rlimit v0.4.0
    Checking users v0.10.0
   Compiling uu_factor v0.0.6 (/home/hbina/git/coreutils/src/uu/factor)
   Compiling uucore_procs v0.0.5 (/home/hbina/git/coreutils/src/uucore_procs)
    Checking uucore v0.0.8 (/home/hbina/git/coreutils/src/uucore)
    Checking uu_base32 v0.0.6 (/home/hbina/git/coreutils/src/uu/base32)
    Checking uu_od v0.0.6 (/home/hbina/git/coreutils/src/uu/od)
    Checking uu_truncate v0.0.6 (/home/hbina/git/coreutils/src/uu/truncate)
    Checking uu_mktemp v0.0.6 (/home/hbina/git/coreutils/src/uu/mktemp)
    Checking uu_rm v0.0.6 (/home/hbina/git/coreutils/src/uu/rm)
    Checking uu_readlink v0.0.6 (/home/hbina/git/coreutils/src/uu/readlink)
    Checking uu_false v0.0.6 (/home/hbina/git/coreutils/src/uu/false)
    Checking uu_fold v0.0.6 (/home/hbina/git/coreutils/src/uu/fold)
    Checking uu_tail v0.0.6 (/home/hbina/git/coreutils/src/uu/tail)
    Checking uu_tac v0.0.6 (/home/hbina/git/coreutils/src/uu/tac)
    Checking uu_hashsum v0.0.6 (/home/hbina/git/coreutils/src/uu/hashsum)
    Checking uu_seq v0.0.6 (/home/hbina/git/coreutils/src/uu/seq)
    Checking uu_date v0.0.6 (/home/hbina/git/coreutils/src/uu/date)
    Checking uu_test v0.0.6 (/home/hbina/git/coreutils/src/uu/test)
    Checking uu_expr v0.0.6 (/home/hbina/git/coreutils/src/uu/expr)
    Checking uu_csplit v0.0.6 (/home/hbina/git/coreutils/src/uu/csplit)
    Checking uu_link v0.0.6 (/home/hbina/git/coreutils/src/uu/link)
    Checking uu_pr v0.0.6 (/home/hbina/git/coreutils/src/uu/pr)
    Checking uu_head v0.0.6 (/home/hbina/git/coreutils/src/uu/head)
    Checking uu_paste v0.0.6 (/home/hbina/git/coreutils/src/uu/paste)
    Checking uu_realpath v0.0.6 (/home/hbina/git/coreutils/src/uu/realpath)
    Checking uu_nl v0.0.6 (/home/hbina/git/coreutils/src/uu/nl)
    Checking uu_unexpand v0.0.6 (/home/hbina/git/coreutils/src/uu/unexpand)
    Checking uu_df v0.0.6 (/home/hbina/git/coreutils/src/uu/df)
    Checking uu_printenv v0.0.6 (/home/hbina/git/coreutils/src/uu/printenv)
    Checking uu_yes v0.0.6 (/home/hbina/git/coreutils/src/uu/yes)
    Checking uu_sort v0.0.6 (/home/hbina/git/coreutils/src/uu/sort)
    Checking uu_ls v0.0.6 (/home/hbina/git/coreutils/src/uu/ls)
    Checking uu_more v0.0.6 (/home/hbina/git/coreutils/src/uu/more)
    Checking uu_du v0.0.6 (/home/hbina/git/coreutils/src/uu/du)
    Checking uu_cp v0.0.6 (/home/hbina/git/coreutils/src/uu/cp)
    Checking uu_dirname v0.0.6 (/home/hbina/git/coreutils/src/uu/dirname)
    Checking uu_printf v0.0.6 (/home/hbina/git/coreutils/src/uu/printf)
    Checking uu_mv v0.0.6 (/home/hbina/git/coreutils/src/uu/mv)
    Checking uu_cat v0.0.6 (/home/hbina/git/coreutils/src/uu/cat)
    Checking uu_sum v0.0.6 (/home/hbina/git/coreutils/src/uu/sum)
    Checking uu_fmt v0.0.6 (/home/hbina/git/coreutils/src/uu/fmt)
    Checking uu_tsort v0.0.6 (/home/hbina/git/coreutils/src/uu/tsort)
    Checking uu_expand v0.0.6 (/home/hbina/git/coreutils/src/uu/expand)
    Checking uu_shuf v0.0.6 (/home/hbina/git/coreutils/src/uu/shuf)
    Checking uu_join v0.0.6 (/home/hbina/git/coreutils/src/uu/join)
    Checking uu_env v0.0.6 (/home/hbina/git/coreutils/src/uu/env)
    Checking uu_tee v0.0.6 (/home/hbina/git/coreutils/src/uu/tee)
    Checking uu_true v0.0.6 (/home/hbina/git/coreutils/src/uu/true)
    Checking uu_shred v0.0.6 (/home/hbina/git/coreutils/src/uu/shred)
    Checking uu_sleep v0.0.6 (/home/hbina/git/coreutils/src/uu/sleep)
    Checking uu_ln v0.0.6 (/home/hbina/git/coreutils/src/uu/ln)
    Checking uu_relpath v0.0.6 (/home/hbina/git/coreutils/src/uu/relpath)
    Checking uu_mkdir v0.0.6 (/home/hbina/git/coreutils/src/uu/mkdir)
    Checking uu_comm v0.0.6 (/home/hbina/git/coreutils/src/uu/comm)
    Checking uu_echo v0.0.6 (/home/hbina/git/coreutils/src/uu/echo)
    Checking uu_cut v0.0.6 (/home/hbina/git/coreutils/src/uu/cut)
    Checking uu_cksum v0.0.6 (/home/hbina/git/coreutils/src/uu/cksum)
    Checking uu_pwd v0.0.6 (/home/hbina/git/coreutils/src/uu/pwd)
    Checking uu_wc v0.0.6 (/home/hbina/git/coreutils/src/uu/wc)
    Checking uu_ptx v0.0.6 (/home/hbina/git/coreutils/src/uu/ptx)
    Checking uu_split v0.0.6 (/home/hbina/git/coreutils/src/uu/split)
    Checking uu_uniq v0.0.6 (/home/hbina/git/coreutils/src/uu/uniq)
    Checking uu_basename v0.0.6 (/home/hbina/git/coreutils/src/uu/basename)
    Checking uu_tr v0.0.6 (/home/hbina/git/coreutils/src/uu/tr)
    Checking uu_rmdir v0.0.6 (/home/hbina/git/coreutils/src/uu/rmdir)
    Checking uu_dircolors v0.0.6 (/home/hbina/git/coreutils/src/uu/dircolors)
    Checking uu_base64 v0.0.6 (/home/hbina/git/coreutils/src/uu/base64)
    Finished dev [unoptimized + debuginfo] target(s) in 8.52s
```

Reply:
nvm, its in nightly